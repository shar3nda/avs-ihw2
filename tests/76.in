g~4oY~)0Ls\u`s$*``~P`jMY5ulJb[U'sMc:V&=gQT&Gd(m,|zr-@7H=-9qc 	_ms/I_M8