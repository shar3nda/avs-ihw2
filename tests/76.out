Lowercase: 20, uppercase: 15